hca/staging_area_validator.py

```diff
@@ -235,19 +235,21 @@ def validate_file_description(self, file_description: str) -> None:
 
     def validate_descriptors_file(self, blob: gcs.Blob) -> None:
         # Expected syntax: descriptors/{metadata_type}/{metadata_id}_{version}.json
-        metadata_type, metadata_file = blob.name.split('/')[-2:]
-        assert metadata_file.count('_') == 1
-        assert metadata_file.endswith('.json')
-        metadata_id, metadata_version = metadata_file.split('_')
+        metadata_type, descriptor_file = blob.name.split('/')[-2:]
+        assert descriptor_file.count('_') == 1
+        assert descriptor_file.endswith('.json')
+
+        metadata_id, descriptor_version = descriptor_file.split('_')
```

Comment:
```suggestion
        metadata_id, descriptor_version = descriptor_file[:-5].split('_')
```
Note: this would require a similar change in `validate_metadata_file` to chop off the `.json` from the filename so it isn't included as part of the version there as well.